static/links.less

```diff
@@ -1,10 +1,13 @@
 @import "ui-variables";
 
 a {
-  cursor: pointer;
   color: @text-color-highlight;
   &:hover {
     color: @text-color-highlight;
     text-decoration: underline;
   }
+
+  &[href^="http"], .link-external {
```

Comment:
I think the `.link-external` needs an `&` in front.

Also that class gets lost a bit being the second selector, but arguably it's the more important one. We don't really have any naming conventions, but how about:

```less
&.link-external,
&[href^="http"] {
  cursor: pointer;
}
```